docs/plugin/alias.md

```diff
@@ -0,0 +1,3 @@
+# Using `Alias`
+Since Docforge can construct the content bundle from multiple repository sources and there could be a need for additional names for certain files,
```

Comment:
Use this for a more extensive documentation https://github.com/gardener/docforge/blob/master/docs/manifests.md#hugo-aliases